scripts/generateSummary.ts

```diff
@@ -0,0 +1,78 @@
+import * as fs from 'fs';
+
+import { getSafeL2SingletonDeployment, getSafeSingletonDeployment } from '../src/safes';
+import { getProxyFactoryDeployment } from '../src/factories';
+import { getDefaultCallbackHandlerDeployment, getCompatibilityFallbackHandlerDeployment, getFallbackHandlerDeployment } from '../src/handler';
+import { getMultiSendDeployment, getMultiSendCallOnlyDeployment, getCreateCallDeployment, getSignMessageLibDeployment } from '../src/libs';
+import axios from 'axios';
+
+export async function generateSummary() {
+
+const response = await axios.get('https://chainid.network/chains.json');
+const ALL_CHAINS: any[] = response.data;
```

Comment:
I had `try-catch` before but removed it because if the request fails the entire script can't work. So there's really no `catch` behavior. 

try-catch could be useful if we could use a local`chains.json` file as a fallback in the `catch` portion.

@rmeissner would you be okay with that?